src/main/java/org/jabref/gui/groups/GroupModeViewModel.java

```diff
@@ -0,0 +1,24 @@
+package org.jabref.gui.groups;
+
+import javafx.scene.Node;
+import javafx.scene.control.Tooltip;
+
+import org.jabref.logic.l10n.Localization;
+
+public class GroupModeViewModel {
+
+    public static Tooltip getUnionIntersectionTooltip(GroupViewMode mode) {
+        if (mode == GroupViewMode.UNION) {
+            return new Tooltip(Localization.lang("Toogle intersection"));
+        }
+        if (mode == GroupViewMode.INTERSECTION) {
+            return new Tooltip(Localization.lang("Toogle union"));
+        }
+        return new Tooltip();
+
+    }
+
+    public static Node getUnionIntersectionGraphic(GroupViewMode mode) {
+        return mode.getIcon().getGraphicNode();
```

Comment:
Don't you agree that a code similar to the above one for the tooltip is better than storing the icon directly in the `groupviewmode`?